.pre-commit-config.yaml

```
@@ -68,6 +73,18 @@ repos:
         args: [--fix]
       # compatible replacement for black
       - id: ruff-format
+  - repo: https://github.com/jumanjihouse/pre-commit-hook-yamlfmt
+    rev: 0.2.3
+    hooks:
+      - id: yamlfmt
+        # ruamel.yaml doesn't line wrap correctly (?) so set width to 1M to avoid issues
+        args: [--mapping, '2', --offset, '2', --sequence, '4', --width, '1000000']
```

What about adding also `--implicit_start` or is there any value that yaml files start with `---`?
```suggestion
        args: [--mapping, '2', --offset, '2', --sequence, '4', --implicit_start, --width, '1000000']
```

---

Good suggestion, only wish we'd found this earlier (https://github.com/conda/infrastructure/pull/915)